internal/testutil/gpgkeys.go

```diff
@@ -0,0 +1,163 @@
+package testutil
+
+import (
+	"log"
+
+	"golang.org/x/crypto/openpgp/packet"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+type Key struct {
+	ID                   string
+	ArmoredPublicKey     string
+	ArmoredPrivateKey    string
+	PrivateKeyPassphrase string
```

Comment:
Do we need a passphrase?

Reply:
We will need a passphrase if we ever want to use a private key with passphrase enabled. But since we are not doing it now, I guess I will just remove it for now.

Reply:
Well, it is my understanding we are never going to handle private keys in chisel. We are only going to do so in testing and the result is the same, a passphrase is only needed to get a plain private key (which is the same as getting the private key directly with no passphrase).